pkg/util/heap/heap_test.go

```diff
@@ -136,8 +136,8 @@ func TestHeap_PushOrUpdate(t *testing.T) {
 	h := New(testHeapObjectKeyFunc, compareInts)
 	h.PushOrUpdate(mkHeapObj("foo", 100))
 	h.PushOrUpdate(mkHeapObj("baz", 20))
-	h.PushOrUpdate(mkHeapObj("foo", 1)) // This behaviors as update.
-	h.PushOrUpdate(mkHeapObj("zab", 8)) // This behaviors as add.
+	h.PushOrUpdate(mkHeapObj("foo", 1)) // This manifests as update.
+	h.PushOrUpdate(mkHeapObj("zab", 8)) // This manifests as add.
```

Comment:
I think that previous comments are correct.

Reply:
behavior is a noun, i think "behavior as" can be replaced with "manifest as", "represent as" or "behave as"

Reply:
I like "behave as" since `PushOrUpdate` is a behavior.